server/src/interfaces/job.interface.ts

```diff
@@ -142,12 +148,20 @@ export interface IEmailJob {
   subject: string;
   html: string;
   text: string;
+  attachments?: Attachment[];
 }
 
 export interface INotifySignupJob extends IEntityJob {
   tempPassword?: string;
 }
 
+export interface INotifyAlbumInviteJob {
+  album: AlbumEntity;
+  guestUser: UserEntity;
+}
+
+export interface INotifyAlbumUpdateJob extends IEntityJob {}
```

Comment:
No need to define a separate interface, just use the `IEntityJob` one.

Reply:
this is WIP